app/scenes/Settings/ImportExport.js

```diff
@@ -199,6 +211,46 @@ function ImportExport() {
           ? `${t("Requesting Export")}…`
           : t("Export Data")}
       </Button>
+      <Subheading>
+        <Trans>Recent exports</Trans>
+      </Subheading>
+      <PaginatedList
+        items={fileOperations.orderedDataExports}
+        fetch={fileOperations.fetchPage}
+        options={{ type: "export" }}
+        renderItem={(item) => (
+          <ListItem
+            key={item.id}
+            title={
+              item.collection ? item.collection.name : t("All collections")
+            }
+            subtitle={
+              <>
+                <Flex>
+                  {stateMapping[item.state]}&nbsp;•&nbsp;
+                  {user.id === item.user.id ? t("You") : item.user.name}{" "}
+                  created&nbsp;
```

Comment:
Missing translation hook here, we should make sure to include the subject to get a proper translation – something like:

```javascript
t(`{{userName}} requested`, { userName: user.id === item.user.id ? t("You") : item.user.name})
```